client/src/components/Players/Players.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { PlayerBriefData } from "../PlayerListTable/PlayerListTable";
+import { tableCategories } from "../../utils";
+import TableCell from "@mui/material/TableCell";
+import PlayerTableRow from "../PlayerTableRow/PlayerTableRow";
+import TableContainer from "@mui/material/TableContainer";
+import { Paper } from "@mui/material";
+import Table from "@mui/material/Table";
+import TableHead from "@mui/material/TableHead";
+import TableRow from "@mui/material/TableRow";
+import TableBody from "@mui/material/TableBody";
+
+const Players = ({playersData}: { playersData: PlayerBriefData[] }) => {
+  const tableHead = tableCategories.map((tableCategories) => {
```

Comment:
```suggestion
  const tableHeadRowCells =
  <>
  <TableCell>Name</TableCell>
  {tableCategories.map(({label}) => <TableCell key={tableCategories.label}  @align="right">{tableCategories.label}</TableCell>)}
  <TableCell align="center">Team</TableCell>
  </>
```
similarly rename tableRowsCells below

Reply:
Create a new component called tabbleHeaRowCells?

Reply:
not necessary, inject like this should be good

Reply:
![image](https://user-images.githubusercontent.com/95100903/200090393-ebe8a338-8426-456b-89f1-ad545e6906d7.png)